packages/client/src/gen/coordinator/index.ts

```diff
@@ -4407,6 +4610,31 @@ export interface UpdatedCallPermissionsEvent {
    */
   user: UserResponse;
 }
+/**
+ *
+ * @export
+ * @interface UserInfoResponse
+ */
+export interface UserInfoResponse {
+  /**
+   *
+   * @type {object}
+   * @memberof UserInfoResponse
+   */
+  Custom: object;
+  /**
+   *
+   * @type {string}
+   * @memberof UserInfoResponse
+   */
+  Image: string;
+  /**
+   *
+   * @type {string}
+   * @memberof UserInfoResponse
+   */
+  Name: string;
+}
```

Comment:
@su225, I'll update these once we get an updated schema (lowercaps props).

Reply:
This is fixed: https://github.com/GetStream/protocol/blob/d3f2446403f533bd624aad41209436964e555586/openapi/video-openapi.yaml#L2482-L2492